phpseclib/Crypt/Base.php

```diff
@@ -44,9 +44,9 @@
  * @package Base
  * @author  Jim Wigginton <terrafrost@php.net>
  * @author  Hans-Juergen Petrich <petrich@tronic-media.com>
- * @access  public
+ * @abstract
```

Comment:
It's in the PEAR coding standards, which phpseclib is based off of:

http://pear.php.net/manual/en/standards.header.php